presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergConfig.java

```diff
@@ -26,6 +29,10 @@
 {
     private IcebergFileFormat fileFormat = PARQUET;
     private HiveCompressionCodec compressionCodec = GZIP;
+    private boolean nativeMode;
+    private String catalogImpl = HiveCatalog.class.getName();
```

Comment:
Iceberg in general uses the pattern of dynamic plugin, users can plugin their own implementations of `Catalog`, `FileIO` and a couple of other important classes. If this is an anti-pattern for Presto, we can move to a Enum type. This feature is mostly oriented for customization of enterprise users, they also need to load in their own jar containing the implementation class during installation time.